newIDE/app/src/EventsBasedBehaviorEditor/EventsBasedBehaviorPropertiesEditor.js

```diff
@@ -225,6 +227,21 @@ export default class EventsBasedBehaviorPropertiesEditor extends React.Component
                           click: () => this._moveProperty(i, i + 1),
                           enabled: i + 1 < properties.getCount(),
                         },
+                        {
+                          label: i18n._(t`Generate functions`),
```

Comment:
Should it be more specific? Generate functions to get and set value for this property?

Reply:
Your second suggestion reads fine I think. And not too long in my opinion

Reply:
Ok, I'll go for "Generate expression and action".